src/Tizen.NUI.Components/Theme/DefaultTheme.cs

```diff
@@ -108,10 +108,15 @@ public HashSet<ExternalThemeKeyList> GetExternalThemeKeyListSet()
 
                 // Picker
                 (new ExternalThemeKeyList(typeof(Picker), typeof(PickerStyle)))
+                    .Add<Size>("/Size", (ViewStyle style, Size value) => ((ViewStyle)style).Size = value)
+                    .AddSelector<float?>("/ItemTextLabel/PixelSize", SetButtonTextPixelSize, ControlState.Selected)
```

Comment:
Please change this as follows.

.AddSelector<float?>("/ItemTextLabel/PixelSize", (ViewStyle style, Selector<float?> value) => ((PickerStyle)style).ItemTextLabel.PixelSize = value, ControlState.Selected)

Reply:
Done